app/views/users/new.html.haml

```diff
@@ -9,7 +15,22 @@
           = f.password_field :password
           = f.text_field :full_name, label: "Full Name"
           = hidden_field_tag :invitation_token, @invitation_token
+        %fieldset.credit_card
+          %span.payment-errors
+          .form-group
+            %label.control-label(for="credit-card-number").col-sm-2 Credit Card Number
+            .col-sm-6
+              %input.form-control#credit-card-number(type="text" data-stripe="number")
+          .form-group
+            %label.control-label(for="security-code").col-sm-2 Security Code
+            .col-sm-6
+              %input.form-control#security-code(type="text" data-stripe="cvc")
+          .form-group
+            %label.control-label.col-sm-2 Expiration
+            .col-sm-3
+              = select_month(Date.today, {add_month_numbers: true},  class: 'form-control', data: {stripe: "exp-month"})
+            .col-sm-2
+              = select_year(Date.today.year, {start_year: Date.today.year, end_year: Date.today.year + 4}, class: 'form-control', data: {stripe: "exp-year"})
         %fieldset.actions.control-group.col-sm-offset-2
           .controls
-            %input(type="submit" value="Sign Up" class="btn btn-default")
-
+            = submit_tag "Sign Up", class: "btn btn-default"
```

Comment:
This could be `f.submit`, you are using `form_for` basically. Does not matter that much.